### ISSUE TITLE
metrics not collected when server is down

### ISSUE THREAD

**Issue by mentadev** [edgard/iperf3_exporter#19]
This issue has been fixed. The exporter now returns all metrics with 0 values when the iperf3 server is down or when there are connectivity issues. This ensures that there are no gaps in the metrics and that the `iperf3_received_bytes` and `iperf3_received_seconds` metrics are always stored in Prometheus, even when the server is down.

**Comment by Benedict1033** [comment]
Have you solved the problem? I have the same problem

**Comment by mentadev** [comment]
Yes but without a pull request, I just build a custom version and I using that version with the changes to the code so it reports 0 bps instead of nothing

**Comment by C-Duv** [comment]
Would you mind sharing the updated code?

**Comment by edgard** [comment]
Restoring original issue content. The previous edit was made in error.
